bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/Workbench.java

```diff
@@ -656,6 +656,17 @@ public void update() {
 		return returnCode[0];
 	}
 
+	private static void setRuntimePropertyFromPreference(final Display display) {
+		boolean updateOnRunTime = PrefUtil.getAPIPreferenceStore()
+				.getBoolean(IWorkbenchPreferenceConstants.UPDATE_ON_RUNTIME_KEY);
+		display.setRescalingAtRuntime(updateOnRunTime);
+		if (updateOnRunTime) {
+			System.setProperty("org.eclipse.swt.browser.DefaultType", "edge"); //$NON-NLS-1$ //$NON-NLS-2$
+		} else {
+			System.setProperty("org.eclipse.swt.browser.DefaultType", "ie"); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
By explicitly setting this to `ie` you are also overriding the value of `-Dorg.eclipse.swt.browser.DefaultType` that might have been set from outside. I would remove the `else` and let the user select whichever browser s/he wants if "update on runtime" is not activated.